src/lib/operators/jit_operator/specialization/jit_code_specializer.cpp

```diff
@@ -12,13 +12,13 @@
 
 #include <queue>
 
-#include "llvm_extensions.hpp"
 #include "jit_runtime_pointer.hpp"
+#include "llvm_extensions.hpp"
 
 namespace opossum {
 
 JitCodeSpecializer::JitCodeSpecializer(JitRepository& repository)
-    : _repository{repository}, _llvm_context{_repository.llvm_context()}, _compiler{_llvm_context} {}
```

Comment:
why was that removed?

Reply:
The block displayed by Github is misleading. Only the compiler was removed because it is now default-constructible.